gcp_variant_transforms/testing/integration/small_tests/merge_option_move_to_calls.json

```diff
@@ -16,6 +16,13 @@
     {
       "query": ["SUM_END_QUERY"],
       "expected_result": {"sum_end": 1283560}
+    },
+    {
+      "query": [
+        "SELECT COUNT(0) AS num_rows FROM {TABLE_NAME} ",
+        "WHERE start_position = 14369"
+      ],
+      "expected_result": {"num_rows": 1}
```

Comment:
For the same `start_position`, you can also count number of calls and verify it.

Reply:
Do you mean `SELECT COUNT(0) AS num_rows FROM {TABLE_NAME} AS t, t.call AS call WHERE start_position = 14369`?
In this case, there is no difference with or without merge, i.e, the expected rows will be 3 for both cases.

Reply:
Correct, they are the same. I meant that in combination with what you already have, i.e., first you are counting that there is only one row in the table with `start_position = 14369` and then you verify that on that single row, there are three calls.

Not a big deal either way, so please feel free to submit as is, if you prefer.

Reply:
Got it! Thanks for the details.